.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,3 +19,13 @@ the linked issue.
 -->
 
 Related to #
+
+#### Things to check
+
+Here are some important things to check on each PR as an author and reviewer.
+
+- [ ] Have you written new tests for your changes?
+- [ ] Have you tested the feature in Simple, Atomic, and self-hosted Jetpack sites?
+- [ ] Are there any TypeScript or console errors?
+- [ ] Are we memoizing when appropriate (for expensive computations)?
+- [ ] Have we sent any new strings for translation ASAP?
```

Comment:
What do you think about adding links to these, so less familiar people can quickly learn what they need to do?

For example, for the testing one, we could link here: https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md

Reply:
Good idea @dpasque. Could you please [check these links](https://github.com/Automattic/wp-calypso/pull/64814/commits/5f45a551c9a3e58912d7a5bb6466dae9adf68665) and see if you have any additional?